frontend/src/components/Checkbox/index.module.css

```diff
@@ -1,26 +1,30 @@
-.inputWrapper {
+.root {
   --size-checkbox: 24px;
 
   position: relative;
   display: flex;
   align-items: center;
 }
 
+.inputWrapper {
+  width: var(--size-checkbox);
```

Comment:
I don't think we need to explicitly set the `width` anymore since `.control` is no longer absolutely positioned. Am I wrong?